### DISCUSSION TITLE
Is it possible to have a rule for QInput validation that calls backend

### DISCUSSION THREAD

**Discussion by chiwa737** [quasarframework/quasar#14921] [General - Components / Directives / etc] [answered]
I have a QInput used in our project, it uses rules for validation, like 'required', etc. But I also need to validate the value from backend as well. 

I am not sure if it is possible to add another **rule** side by side that has validation logic from backend.

**Comment by metalsadman** [comment, marked as answer]
Use an async rule

**Reply by chiwa737** [reply]
Thank you for pointing the direction!, it works
